### PR TITLE
create new connector for mokosmart

### PR DESCRIPTION
**Pull request by mokotechnology**
## Decoder Description

Explain the functionality of your decoder or any other relevant information.

## Type of change

- [x] Adding a new Decoder of Connector
- [ ] Adding a new Decoder of Network
- [ ] Update or fixing an issue in existing Decoder

## Decoder Information and Payload to test and review

- [x] Documentation of the hardware or protocol:
- [x] Payload of example the test the decoder:

## Checklist for Adding a New Decoder

- [x] Created a new folder under `./decoders/network/` or `./decoders/connector/` with the name of your decoder.
- [x] Added a `network.jsonc` or `connector.jsonc` file that follows the structure defined in `./schema/`.
- [x] Created version folders and added `manifest.jsonc` files for each version.
- [x] Followed the folder structure guidelines for manufacturer and sensor/device model.
- [ ] The code has unit test and it's in TypeScript.

## Additional Notes

Please add any other information that you think is important.

### CODE REVIEW

File: decoders/connector/mokosmart/LW006/v1.0.0/payload.js [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+        var start = i;
+        var end = i + 2;
+        var data = parseInt("0x" + hex.substring(start, end));
+        datas.push(data);
+    }
+    return datas;
+
+}
+function addPayloadArray(variable, value) {
+    return payload.push({ variable: variable, value: value, group: String(payload_raw.group), serie: String(payload_raw.serie), });
```
Comment (mateuscardosodeveloper) [inline decoders/connector/mokosmart/LW006/v1.0.0/payload.js:390]: `serie` is deprecated. This payload parser must be updated to use `group` instead

File: decoders/connector/mokosmart/LW008-MT/v1.0.0/payload.js [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+function int8(byte) {
+    var sign = byte & (1 << 7);
+    if (sign) {
+        return 0xFFFFFF00 | byte;
+    }
+    return byte;
+}
+
+function addPayloadArray(variable, value) {
+    return payload.push({ variable: variable, value: value, group: String(payload_raw.group), serie: String(payload_raw.serie), });
```
Comment (mateuscardosodeveloper) [inline decoders/connector/mokosmart/LW008-MT/v1.0.0/payload.js:346]: `serie` is deprecated. This payload parser must be updated to use `group` instead

**Review by mateuscardosodeveloper — CHANGES_REQUESTED** [review]

File: decoders/connector/mokosmart/LW006/v1.0.0/payload.js [hunk 3]
```diff
@@ -0,0 +1,405 @@
+var turnOffMode = ["Bluetooth", "LoRa", "Button", "Low Battery"];
```
[the remark this answers is not in the document]
Comment (mokotechnology) [reply decoders/connector/mokosmart/LW006/v1.0.0/payload.js]: Hi, I have added the test for parser, pls review again. thanks

File: decoders/connector/mokosmart/LW008-MT/v1.0.0/payload.js (hunk 2 above)
Comment (mokotechnology) [reply decoders/connector/mokosmart/LW008-MT/v1.0.0/payload.js]: Hi, I have changed according to your instruction, pls review again, thanks

File: decoders/connector/mokosmart/LW006/v1.0.0/payload.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (mateuscardosodeveloper) [reply decoders/connector/mokosmart/LW006/v1.0.0/payload.js]: Hi @mokotechnology, your unit tests aren't passing. Also need to pass your payload.js to TypeScript with type annotation. Thank you.

[the remark this answers is not in the document]
Comment (mokotechnology) [reply decoders/connector/mokosmart/LW006/v1.0.0/payload.js]: Hi, I have fixed the bugs, pls review again.thanks

[the remark this answers is not in the document]
Comment (mokotechnology) [reply decoders/connector/mokosmart/LW006/v1.0.0/payload.js]: Hi, I have convert the payload.js files to TypeScript,pls review again.thanks

File: decoders/connector/mokosmart/LW006/assets/logo.png
Comment (Freddyminu) [inline decoders/connector/mokosmart/LW006/assets/logo.png:1]: This must be a .png (transparent background)

**Review by mateuscardosodeveloper — APPROVED** [review]

File: decoders/connector/mokosmart/LW006/description.md [hunk 4]
```diff
@@ -0,0 +1 @@
+LW006 Smart Badge, the smart badge that combines LoRaWAN network transmission with BLE,WIFI, and GPS positioning technologies. This combination allows for reliable indoor and outdoor positioning, making it ideal for ensuring staff safety, emergency alarm capabilities, and effective personnel management.
```
Comment (Freddyminu) [inline decoders/connector/mokosmart/LW006/description.md:1]: This must be one short sentence quickly stating factual information regarding the sensor. This is one of the fields that will be searched by the Connector page.

File: decoders/connector/mokosmart/LW008-MT/assets/logo.png
Comment (Freddyminu) [inline decoders/connector/mokosmart/LW008-MT/assets/logo.png:1]: This must be a .png (transparent background)

File: decoders/connector/mokosmart/LW008-MT/description.md [hunk 5]
```diff
@@ -0,0 +1 @@
+LW008-MT series are small size LoRaWAN Trackers that can be used for indoor and outdoor positioning, for building construction, logistics tracking or other asset tracking.
```
Comment (Freddyminu) [inline decoders/connector/mokosmart/LW008-MT/description.md:1]: Example: "Compact trackers for indoor and outdoor positioning over LoRaWAN™"

**Review by Freddyminu — DISMISSED** [review]

File: decoders/connector/mokosmart/LW006/assets/logo.png
Comment (Freddyminu) [inline decoders/connector/mokosmart/LW006/assets/logo.png:1]: This must be a .png (transparent background)

**Review by Freddyminu — CHANGES_REQUESTED** [review]

File: decoders/connector/mokosmart/LW006/assets/logo.png
Comment (mokotechnology) [reply decoders/connector/mokosmart/LW006/assets/logo.png:1]: Hi, I have update the log and description,pls review again.thanks

**Review by Freddyminu — APPROVED** [review]

**Pull request merged**